### ISSUE TITLE
Verifier: unpacking files

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent#225]
This is a part of the implementation of the #193 blueprint. This task depends on #221.

In this part of code you can assume that you have access to two .zip achives: an archive with the sources of a Blender project and another with the rendering result submitted by the provider (#221).

### Sequence of operation
- Verifier unpacks the archive with project source.
    - If unpacking fails (out of disk space, not an archive, etc.) verifier sends `verification_result` describing the error to the work queue and verification ends.
    - Make sure that unpacking does not require the whole file to be loaded into memory. Memory requirements should be independent of file size.
- Verifier runs blender (separate issue).
- Verifier unpacks the archive with provider's rendering result.
    - Same as above: verification ends if unpacking fails; watch your memory usage.

**Comment by rwrzesien** [comment]
@cameel 
> Verifier unpacks the archive with project source.
> Verifier unpacks the archive with provider's rendering result.

Would that be a problem if I do those steps one after another ?

**Comment by cameel** [comment]
No, but you will need a little less storage with no real downsides if you wait with unpacking the results until you're finished with rendering because can can delete the source package immediately after rendering.

Actually, now that I think about it, we should do the some with download. We don't need to download the result until we're finished with rendering. Or we could do this while we're rendering.
